File: Arcade/Intro/JavaScript/34_extractEachKth.js
function extractEachKth(inputArray, k) {
    var newArr = [];
    for (var i = 0; i < inputArray.length; i++) {
        if ((i + 1) % k !== 0) {
            newArr.push(inputArray[i]);
        }
    }
    return newArr;
}

File: Arcade/Intro/C++/34_extractEachKth.cpp
std::vector<int> extractEachKth(std::vector<int> inputArray, int k) {
    vector<int> temp;
    for(int i=0;i<inputArray.size();i++) {
        if((i+1)%k!=0) temp.push_back(inputArray[i]);
    }
    return temp;
}
